repository/d.json

```diff
@@ -77,6 +77,18 @@
 				}
 			]
 		},
+		{
+			"name": "DailyOrganizer",
+			"details": "https://github.com/mremallin/DailyOrganizer",
+			"labels": ["productivity"],
+			"description": "A simple note organizer for Sublime Text.",
```

Comment:
If you remove the description here it'll take that of the GitHub repo automatically, much easier to maintain.

Reply:
Done